src/TestFramework/SafeQuery.php

```diff
@@ -33,12 +33,25 @@
 
 declare(strict_types=1);
 
-namespace Infection\Config\Guesser;
+namespace Infection\TestFramework;
 
-/**
- * @internal
- */
-interface Guesser
+use DOMElement;
+use DOMNodeList;
+use DOMXPath;
+use Webmozart\Assert\Assert;
+
+trait SafeQuery
```

Comment:
The method this trait provides was already implemented in two classes, so i've extracted it into a trait instead, as this PR adds it in more places